relationships/synthesis/EXT-SERVICE-to-INFRA-AWSMQTOPIC.yml

```diff
@@ -0,0 +1,22 @@
+relationships:
+  - name: extServiceCallsAwsMqTopic
+    version: "1"
+    origins:
+      - OpenTelemetry
+    conditions:
```

Comment:
These conditions are too open. You need to introduce something more specific to AWS MQ (or ActiveMQ).